tools/deploy-transfer-ownership-calibnet.sh

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#####################################
+# Environment variables & defaults  #
+#####################################
+
+: "${FIL_CALIBNET_RPC_URL:?FIL_CALIBNET_RPC_URL not set. Please export it and rerun.}"
+: "${FIL_CALIBNET_PRIVATE_KEY:?FIL_CALIBNET_PRIVATE_KEY not set. Please export it and rerun.}"
+: "${NEW_OWNER:?NEW_OWNER not set. Please export it and rerun.}"
+
+
+CHAIN_ID="${CHAIN_ID:-314159}"
+COMPILER_VERSION="${COMPILER_VERSION:-0.8.22}"
+
+#####################################
+# 1. Create INIT_DATA               #
+#####################################
+echo "Generating calldata for initialize(uint256) with argument 150 ..."
+INIT_DATA=$(cast calldata "initialize(uint256)" 150)
+echo "INIT_DATA = $INIT_DATA"
+echo
+
+#####################################
+# 1. Get deployer address           #
+#####################################
+echo "Deriving deployer address from private key ..."
+DEPLOYER_ADDRESS=$(cast wallet address "$FIL_CALIBNET_PRIVATE_KEY")
+echo "Deployer address: $DEPLOYER_ADDRESS"
+echo
+
+#####################################
+# 2. Deploy PDPVerifier contract    #
+#####################################
+echo "Deploying PDPVerifier contract ..."
+DEPLOY_OUTPUT_VERIFIER=$(
+  forge create \
+    --rpc-url "$FIL_CALIBNET_RPC_URL" \
+    --private-key "$FIL_CALIBNET_PRIVATE_KEY" \
+    --chain-id "$CHAIN_ID" \
+    --compiler-version "$COMPILER_VERSION" \
+    --json \
+    src/PDPVerifier.sol:PDPVerifier
+)
+
+# Extract the deployed address from JSON output
+PDP_VERIFIER_ADDRESS=$(echo "$DEPLOY_OUTPUT_VERIFIER" | jq -r '.deployedTo')
```

Comment:
I'm having trouble parsing this on both OSX and linux.  There is no deployedTo field in my output json.  Have you gotten it working since making this change?